figures/scripts/supp-figureS2.R

```diff
@@ -0,0 +1,39 @@
+# Stephanie J. Spielman for CCDL, 2022
```

Comment:
I think this snuck in. Always look at the diff! 😸

Reply:
Ahh!! my bad.